libbpf-tools/opensnoop.bpf.c

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2019 Facebook
+// Copyright (c) 2020 Netflix
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include "opensnoop.h"
+
+#define TASK_RUNNING 0
+
+#define BPF_F_INDEX_MASK		0xffffffffULL
+#define BPF_F_CURRENT_CPU		BPF_F_INDEX_MASK
```

Comment:
this shouldn't be needed as well, but requires a bunch of work in BTF support for enum, so it has to be copy/pasted for now. It's on my TODO list as well...